<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleApplyMain</class>
 <widget class="QWidget" name="SampleApplyMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>710</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1182</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1182</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_up" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1180</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1180</width>
     <height>180</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lab_num">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>患者编号</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_num">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 302px;
height: 58px;
background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lab_type">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 42px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_type" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>PingFang SC</family>
            <pointsize>-1</pointsize>
            <weight>62</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>急诊</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_emergency">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>103</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/beijing/emergency_none.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lab_barcode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 42px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>样本条码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_barcode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 302px;
height: 58px;
background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lab_type_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 65px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>样本管</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_type_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>353</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_switc_page_up" native="true">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>185</y>
     <width>171</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_switc_page_up{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>61</horstretch>
      <verstretch>51</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
image: url(:/beijing/switch_left.png);
border-top-left-radius: 30px;
border-bottom-left-radius: 30px;
border-top-right-radius: 0px; 
border-bottom-right-radius: 0px;
border-left:1px solid rgb(155,159,160);
border-top:1px solid rgb(155,159,160);
border-bottom:1px solid rgb(155,159,160);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_msg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>16</y>
      <width>50</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 402px;
height: 58px;
background-color: #FFFFFF;
border: 1px solid rgb(155,159,160);

border-left: none; 
border-right: none;         
border-top: 1px solid rgb(155,159,160); 
border-bottom: 1px solid rgb(155,159,160);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #74caf6;
line-height: 35px;
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_right">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>16</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>61</horstretch>
      <verstretch>51</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
image: url(:/beijing/switch_right.png);
border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 30px; 
border-bottom-right-radius: 30px;
border-right:1px solid rgb(155,159,160);
border-top:1px solid rgb(155,159,160);
border-bottom:1px solid rgb(155,159,160);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>edit_msg</zorder>
   <zorder>btn_right</zorder>
   <zorder>btn_left</zorder>
  </widget>
  <widget class="QWidget" name="widget_down" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>190</y>
     <width>991</width>
     <height>280</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>991</width>
     <height>280</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>990</width>
     <height>280</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_switc_page_down" native="true">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>458</y>
     <width>171</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_switc_page_down{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLineEdit" name="edit_msg_down">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>50</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 402px;
height: 58px;
background-color: #FFFFFF;
border: 1px solid rgb(155,159,160);

border-left: none; 
border-right: none;         
border-top: 1px solid rgb(155,159,160); 
border-bottom: 1px solid rgb(155,159,160);

font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #74caf6;
line-height: 35px;
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_right_down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>61</horstretch>
      <verstretch>51</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
image: url(:/beijing/switch_right.png);
border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 30px; 
border-bottom-right-radius: 30px;
border-right:1px solid rgb(155,159,160);
border-top:1px solid rgb(155,159,160);
border-bottom:1px solid rgb(155,159,160);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_left_down">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>61</horstretch>
      <verstretch>51</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
image: url(:/beijing/switch_left.png);
border-top-left-radius: 30px;
border-bottom-left-radius: 30px;
border-top-right-radius: 0px; 
border-bottom-right-radius: 0px;
border-left:1px solid rgb(155,159,160);
border-top:1px solid rgb(155,159,160);
border-bottom:1px solid rgb(155,159,160);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_down_2" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>460</y>
     <width>991</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>991</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>991</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
